packages/lodestar/src/network/encoders/snappy-frames/uncompress.ts

```diff
@@ -0,0 +1,101 @@
+import BufferList from "bl";
+import {uncompress} from "snappyjs";
+import {IDecompressor} from "../interface";
+
+const IDENTIFIER = new Buffer([
+  0x73, 0x4e, 0x61, 0x50, 0x70, 0x59
+]);
+
+export class SnappyFramesUncompress implements IDecompressor {
```

Comment:
Looks great. Should we enhance `snappyjs` with this?

Reply:
I don't think this implementation will be usefull to snappyjs as most common scenario should be to just yield data everytime you uncompress something and I don't know how to incorporate async generator like that in our case. But yeah, I would love to create separate library it-snappyjs that would use async iterators to compress and decompress using framing format